net/nut/Config.in

```diff
@@ -97,4 +23,10 @@
 		help
 			If you have a UPS you can connect to via SNMP, select this.
 		default n
-		
+
+	config NUT_DRIVER_SERIAL
+		depends on PACKAGE_nut
+		bool "Build with suport for serial drivers"
```

Comment:
Does build fail for you without libpthread dependency?  I'm not sure it is actually required (I've had no issues but I primarily use MUSL based builds so maybe GLIBC requires it?

And typo noted.